pkg/actions/karpenter/create.go

```diff
@@ -74,11 +81,6 @@ func (i *Installer) Create() error {
 		return fmt.Errorf("failed to save the identity config: %w", err)
 	}
 
-	instanceProfileName := fmt.Sprintf("eksctl-%s-%s", builder.KarpenterNodeInstanceProfile, i.Config.Metadata.Name)
-	if i.Config.Karpenter.DefaultInstanceProfile != nil {
-		instanceProfileName = aws.StringValue(i.Config.Karpenter.DefaultInstanceProfile)
-	}
-
```

Comment:
not seeing any changes in the test to reflect this :thinking:

Reply:
what do you mean? I deleted a BUNCH of stuff. :)

Reply:
Also, this just moved, the thing remained but it is on the top

Reply:
ohh sorry I thought it was actually changing thing. Git diffs are hard to read sometimes mb

Reply:
Indeed! It can be difficult to see especially when things move around. :/